recipes/treemaker/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
```

Comment:
Please see this section: https://bioconda.github.io/linting.html#setup-py-install-args
and change this to `$PYTHON setup.py install --single-version-externally-managed --record=record.txt`